node/pkg/driver/device_connectivity/device_connectivity_helper_scsigeneric.go

```diff
@@ -589,3 +697,71 @@ func (o GetDmsPathHelperGeneric) WaitForDmToExist(volumeUuid string, volumeNguid
 	}
 	return "", err
 }
+
+func (o GetDmsPathHelperGeneric) GetDmObjectsByIdentifier(wantedDmObjects []string, outputToCheckIn string) map[string]bool {
+	dmObjects := make(map[string]bool)
```

Comment:
`Objects` is too general. how about `fieldValues`?

Reply:
done

Reply:
I still see (dm)`Object` in the PR diff

Reply:
I am not sure that that field value is good [here](https://github.com/IBM/ibm-block-csi-driver/blob/7a9d7dfef1aca9ffc69211291f5dc9ced31512be/node/pkg/driver/device_connectivity/device_connectivity_helper_scsigeneric.go#L701), what do you think?

Reply:
changed all of the `object` in my PR except the ones that I mentioned in my previous comment in this conversation

Reply:
I think it's better than "object", especially since both the field value and the identifier are referred to as "DM object" in said log message, which is confusing.
and I don't have a better idea than (dm) field value